CMakeLists.txt

```
@@ -23,6 +23,9 @@ include(CompilerSettings)
 include(BuildInfoGenerator)
 
 # import dependencies
+include(ProjectGroupSig)
+include(ProjectCryptopp)
```

```suggestion
hunter_add(cryptopp)
```
use hunter instead of project.